README.md

```diff
@@ -57,6 +57,25 @@ This will start the development server on your local machine and allow you to st
    ```
    npm run new
    ```
+4. Run linter by the following command:
+   
+   ```
+   npm run lint
+   ```
+
+5. Run all test using the following command.
+
+   ```
+   npm run test
+   ``` 
+   Note : Since packages have interdependency, you need to run `dev` script atleast once before running `lint` and `test` scripts globally.
```

Comment:
Please add this as a markdown _comment_ with the `>` syntax. Use comments elsewhere as well, if applicable.

Also, I think you can then either omit the word "Note" altogether, or at least remove the whitespace _before_ the colon.